features/manage-position/YourPosition.tsx

```diff
@@ -115,49 +115,64 @@ const YourPosition = () => {
-        <Status>
-          <Label>Collateral supplied: </Label>
-          {`${_collateral} ${_collSymbol}`}
-        </Status>
-        <Status>
-          <Label>Tokens outstanding: </Label>
-          {`${_tokens} ${_tokenSymbol}`}
-        </Status>
-        <Status>
-          <Label>
-            Estimated identifier price (
-            <Link href={pricefeedUrl} target="_blank" rel="noopener noreferrer">
-              Coinbase Pro
-            </Link>
-            ):{" "}
-          </Label>
-          {_latestPrice}
-        </Status>
-        <Status>
-          <Label>(CR) Collateralization ratio:</Label>
-          {` ${pricedCR} (${_collSymbol} / ${_tokenSymbol})`}
-        </Status>
-        <Status>
-          <Label>(GCR) Global collateralization ratio:</Label>
-          {` ${pricedGCR} (${_collSymbol} / ${_tokenSymbol})`}
-        </Status>
-        <Status>
-          <Label>Liquidation Requirement:</Label>
-          {` ${collReqFromWei}`}
-        </Status>
-        <Status>
-          <Label>Liquidation Price:</Label>
-          {` ${liquidationPrice} (${priceIdUtf8})`}
-        </Status>
-        <Status>
-          <Label>Collateral pending/available to withdraw:</Label>
-          {` ${_withdrawAmt} ${_collSymbol}`}
-        </Status>
-        <Status>
-          <Label>Pending withdrawal request:</Label>
-          {` ${_pendingWithdraw}`}
-        </Status>
+        <Grid container spacing={4}>
+          <Grid item md={6} xs={12}>
+            <Typography variant="h5">Your Position</Typography>
+            <Status>
+              <Label>Collateral supplied: </Label>
+              {`${_collateral} ${_collSymbol}`}
+            </Status>
+            <Status>
+              <Label>Tokens outstanding: </Label>
+              {`${_tokens} ${_tokenSymbol}`}
+            </Status>
+            <Status>
+              <Label>Collateral ratio (CR):</Label>
+              {` ${pricedCR}`}
+            </Status>
+            <Status>
+              <Label>Liquidation price:</Label>
+              {` ${liquidationPrice} (${priceIdUtf8})`}
+            </Status>
+
+            <Status>
+              <Label>Pending withdrawal request:</Label>
+              {` ${_pendingWithdraw}`}
+            </Status>
+            {_pendingWithdraw == "available" && (
+              <Status>
+                <Label>Collateral available to Withdraw:</Label>
+                {` ${_withdrawAmt}`}
+              </Status>
+            )}
+          </Grid>
+          <Grid item md={6} xs={12}>
+            <Typography variant="h5">Contract Info</Typography>
+            <Status>
+              <Label>
+                Identifier price(
+                <Link
+                  href={pricefeedUrl}
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  Coinbase
+                </Link>
+                ):{" "}
+              </Label>
+              {Number(_latestPrice).toFixed(4)}
```

Comment:
If you think 4 decimals is better than 6, just change the `_latestPrice` that is passed into `renderComponent` instead of manipulating down here. I really want to keep our TSX components as stateless as possible

Reply:
I have decreased text size on `xs` screens which helped a lot.